Admin/AbstractAdmin.php

```diff
@@ -2873,7 +2873,8 @@ public function getDashboardActions()
     final public function getSearchResultLink($object)
     {
         foreach ($this->searchResultActions as $action) {
-            if ($this->hasRoute($action) && $this->isGranted(strtoupper($action), $object)) {
+            $role = 'show' === $action ? 'VIEW' : strtoupper($action);
```

Comment:
Maybe we should rename the action or the role name, so we don't need this mapping?